docs/source/notebooks/GP-Circular.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<ul><li><code>vars</code>  is deprecated, better to use <code>var_names</code>  instead</li><li>maybe let's not call the object <code>trace</code>,  as this is usually reserved to posterior samples, not posterior predictive -- could be confusing to readers</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I've used varnames, but missed the `trace` suggestion